xdcr/cbs_xdcr_test.go

```diff
@@ -32,7 +33,78 @@ func TestCBSXDCR(t *testing.T) {
 	toBucket, err := base.AsGocbV2Bucket(bucket2)
 	require.NoError(t, err)
 
-	xdcr, err := NewCouchbaseServerXDCR(ctx, fromBucket, toBucket)
+	xdcr, err := NewCouchbaseServerXDCR(ctx, fromBucket, toBucket, serverMobileOff)
+	require.NoError(t, err)
+	// set filter for sync docs as mobile xdcr is not on
+	xdcr.filter = fmt.Sprintf("NOT REGEXP_CONTAINS(META().id, \"^%s\") OR REGEXP_CONTAINS(META().id, \"^%s\")", base.SyncDocPrefix, base.Att2Prefix)
```

Comment:
should this be based on what version of couchbase server we are running? I don't think we should test this feature with a filter expression at all.

I would test this based on versions of server whether the _sync docs are going to get copied.

If we set the filter option, we aren't going to see the results of {{-mobile}} setting.

Reply:
I think we either remove the test altogether or leave as is until we start running integration tests against 7.6.1. The idea of me doing this was so we can have a test that will keep testing the XDCR functionality until we can properly run tests against 7.6.1. I'm filing a ticket to get 7.6.1 in Jenkins so can link this test to that so once we can run tests in Jenkins we can remove it.

Reply:
I think just change this test to make sure XDCR is working differently in different server version? We should be able to expose `GTestBucketPool.supportsMobileXDCR` ? 

Maybe even have it assert on the existence of sync docs with < 7.6.1, and not otherwise.

I see the xdcr package as mostly general still which we could use to test a unidirectional DR XDCR issue.